django_minio_backend/models.py

```diff
@@ -248,7 +249,8 @@ def url(self, name: str):
         :return: (str) URL to object
         """
         if self.is_bucket_public:
-            return f'{self.base_url_external}/{self.bucket}/{name}'
+            base_url = self.client._base_url.build("GET", self.__MINIO_REGION).geturl()
+            return f'{base_url}/{self.bucket}/{name}'
```

Comment:
`.geturl()` returns a url with a trailing `/` so we should return `f'{base_url}{self.bucket}/{name}'` instead to avoid duplicating slashes

Reply:
Made the changes 👍